.github/workflows/issue-manager.yml

```
@@ -0,0 +1,47 @@
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  issue-manager:
+    if: github.repository_owner == 'khulnasoft'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(pick(github, ['event_name', 'repository', 'issue.number'])) }}
```

actionlint (1.7.4)

unexpected token "[" while parsing variable access, function call, null, bool, int, float or string. expecting "IDENT", "(", "INTEGER", "FLOAT", "STRING"

(expression)